remoteappmanager/cli/tests/test_remoteapprest.py

```diff
@@ -141,6 +141,18 @@ def test_app_start(self):
             self.assertEqual(mock_post.call_args[0][1],
                              json.dumps({"mapping_id": "1"}))
 
+            self._remoteapprest("app start 1 --startupdata=/test")
```

Comment:
Is there anyway we can test the situation where the application policy for `1` does not allow `startupdata` to be assigned / is not supported?